Sources/Action.swift

```diff
@@ -168,6 +171,74 @@ public final class Action<Input, Output, Error: Swift.Error> {
 		}
 	}
 
+	/// Create an `Action` that wraps an inner `Action` with an additional availability
+	/// condition applied.
+	///
+	/// - parameters:
+	///   - inner: The inner `Action`.
+	///   - isEnabled: An optional additional `Action` availability condition.
+	public convenience init<Enabled: PropertyProtocol>(
+		inner: Action<Input, Output, Error>,
+		enabledIf isEnabled: Enabled
+	) where Enabled.Value == Bool {
```

Comment:
The general idea makes sense, but this API may need some refinement. If nothing else, `inner` should probably have no name externally. But I'm not totally convinced that these should be `init`s.